development/webpack/webpack.base.config.js

```
@@ -17,9 +17,8 @@ class BuildDoneNotifyPlugin {
       if (IS_EAS_BUILD) {
         exit(0);
       } else {
-        const msg = `OneKey Build at ${new Date().toLocaleTimeString()}, completed in ${
-          (compilation.endTime - compilation.startTime) / 1000
-        }s`;
+        const msg = `OneKey Build at ${new Date().toLocaleTimeString()}, completed in ${(compilation.endTime - compilation.startTime) / 1000
```

ESLint

[error] Insert `⏎··········`

(prettier/prettier)